OSNetworkRequirementChecker-HLApp/src/dns-helper.c

```diff
@@ -0,0 +1,519 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This code appears to be based off an older version of the DNSServiceDiscovery sample from the azure-sphere-samples repository - it contains a bug where misleading results will be shown if there is more than one message is receieved. Please see the changes in [this PR](https://github.com/Azure/azure-sphere-samples/pull/196/files#diff-d889dd24f89bdfb65d958c52bf2b71d51b9055f9c707889b2d7573877b38df94) for a fix.

Reply:
I see. thanks for the pointer! updated.